higlass/server.py

```diff
@@ -24,17 +27,20 @@
 __all__ = ["Server"]
 
 OS_NAME = platform.system()
+OS_TEMPDIR = tempfile.gettempdir()
 
-# Disable annoying flask logs
+
+# Disable annoying logs from werkzeug server
 log = logging.getLogger("werkzeug")
 log.setLevel(logging.ERROR)
 log.disabled = True
+
 # The following line is also needed to turn off all debug logs
 os.environ["WERKZEUG_RUN_MAIN"] = "true"
 
 
-def create_app(tilesets, name, log_file, log_level, file_ids, fuse=None):
-    app = Flask(__name__)
+def create_app(name, tilesets, fuse=None):
```

Comment:
Is that API change public? I.e., does a user need to care about it? If not please ignore my comment.

Reply:
So far, nobody uses the app factory, except maybe me. Going forward, it is pretty well encapsulated in `Server` anyway.

Reply:
Cool thanks. Was just wondering if this has a big impact or not.